cmd/statusd/utils.go

```diff
@@ -1414,6 +1418,29 @@ func testJailFunctionCall(t *testing.T) bool {
 	return true
 }
 
+func testNodeOffline(t *testing.T) bool {
+
+	StopNode()
+	//time to sync
+	time.Sleep(5 * time.Second)
```

Comment:
Ah, sorry, I've commented it thinking it was about blockchain synchronisation.
In this case yes, as I know, there's no way to do it more reliably except for making `StopNode` signal stop node synchronously and introducing a new signal `StopNodeAsync` which does it asynchronously. Anyway, in this case, I'd add a 10-second delay and proposed that change. Which is absolutely logical in my opinion.